@here/harp-omv-datasource/lib/OmvDataSource.ts

```diff
@@ -212,6 +212,7 @@ export class OmvDataSource extends TileDataSource<OmvTile> {
         });
 
         this.cacheable = true;
+        this.addGroundPlane = true;
```

Comment:
I think this should be configurable (i.e. in the OmvDataSourceOptions). Especially for hybrid usecase.

Reply:
Which hybrid use cases do you mean? Are you referring to some DataSource which extends OmvDataSource and includes some raster tiles?

Reply:
Done